api/src/main/java/net/kyori/adventure/sound/Sound.java

```diff
@@ -274,4 +303,112 @@ interface Emitter {
       return SoundImpl.EMITTER_SELF;
     }
   }
+
+  /**
+   * A builder for sound instances.
+   *
+   * <p>Type is required, all other options are optional.</p>
+   *
+   * @since 4.12.0
+   */
+  interface Builder extends AbstractBuilder<Sound> {
```

Comment:
Do we not normally instead extend Buildable/Builder rather than abstract builder?

Reply:
AbstractBuilder is a new interface replacing `Builder` -- me and @kashike are discussing a new pattern for builders that replaces `Built#toBuilder()` with `Built.built(existing)`, though we're unsure about its exact form.

Reply:
Sounds dope to me then!